bindings/python/iota_sdk/types/block/metadata.py

```diff
@@ -32,10 +32,10 @@ class BlockState(Enum):
     """Describes the state of a block.
 
     Attributes:
-        Pending: Stored but not confirmed.
-        Accepted: TODO
-        Confirmed: Confirmed with the first level of knowledge.
-        Finalized: Included and can no longer be reverted.
+        Pending: Stored but not accepted/confirmed.
+        Accepted: Valid block referenced by some validators.
```

Comment:
No better definition than "some"?

Reply:
I think it's fine if we compare it to Confirmed we understand that it's less than 2/3 but I can add it if you want

Reply:
I just would myself understand what exactly this means, is a single reference from a single validator already enough to have it accepted?

Reply:
I would say so, yes. I can draft and wait for a better comment 🤷🏻